### ISSUE TITLE
AttributeError when using `Object.get_app_link()`

### ISSUE THREAD

**Issue by bsoyka** [tylertian123/pyryver#10]
Just got this error when using `Task.get_app_link()`:

```
  File ".../pyryver/objects.py", line 278, in get_app_link
    return f"https://{self._ryver.org}.ryver.com/#{self.get_type()}/{self.get_id()}"
                      ^^^^^^^^^^^^^^^
AttributeError: 'Ryver' object has no attribute 'org'. Did you mean: '_org'?
```

Looks to me like the suggestion given is right, and that `org` just needs to change to `_org` in the `Object` class here:

https://github.com/tylertian123/pyryver/blob/51623fa339b028ce7bd54905222128b95fbfd211/pyryver/objects.py#L278

Happy to submit a quick PR for this. Let me know!

**Comment by mincrmatt12** [comment]
Fixed in `v0.4.0.post1`.

Also note that as of right now there isn't much maintenance happening on `pyryver`; I only committed this because it was so trivial.

**Comment by bsoyka** [comment]
Awesome, appreciate the quick fix!

And yeah, totally understandable. Not like there's much to keep up with anyways in terms of new Ryver features. 🤣
